.storybook/config.js

```diff
@@ -0,0 +1,21 @@
+import { configure } from '@storybook/vue';
+import { action } from '@storybook/addon-actions';
+
+import Vue from 'vue';
+import BootstrapVue from 'bootstrap-vue'
+
+import 'bootstrap/dist/css/bootstrap.css'
```

Comment:
This appears to result in the Bootstrap CSS being included inline in the server's HTML response for every page. Is that what we want, as opposed to linking to it on a CDN?